deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -397,6 +397,28 @@ cephFileSystems:
         # in hyperconverged settings where the volume is mounted on the same node as the osds.
         csi.storage.k8s.io/fstype: ext4
 
+cephFileSystemVolumeSnapshotClass:
+  enabled: true
+  name: ceph-filesystem-vsc
+  isDefault: true
+  deletionPolicy: Delete
+  # see https://github.com/rook/rook/blob/master/Documentation/ceph-csi-snapshot.md#cephfs-snapshots for available configuration
+  parameters:
+    # The secrets contain Ceph admin credentials.
+    csi.storage.k8s.io/snapshotter-secret-name: rook-csi-cephfs-provisioner
```

Comment:
Can we add these parameters to the template? We know the provisioner name and namespace, right? So we can create these in the template instead of in the values.yaml. When this chart was created we leaned towards putting everything in values.yaml, but now I'm leaning more towards more in the template if possible.

Reply:
done. moved storageclass secrets also to templates as its more appropriate places for secrets.